README.md

```diff
@@ -1,50 +1,41 @@
 # HTML5 Semantic Elements
 
-## Overview
+## Problem Statement
 
-In this lesson we will learn about semantic elements, their purpose, and best practices for using them.
+We have `div` elements to organize and label sections of HTML. But too many `div` tags makes our HTML document look cluttered and confusing. Even when we identify or classify them with specific names, it's a lot to keep track of. It would be clearer for both developers and browsers if we could use more descriptive HTML elements to arrange our content. That's where semantic elements come in.
 
```

Comment:
I'm not sure I'm loving this as a problem statement in the particular Flatiron style. What about something like this?
`div` tags allow us to organize and label sections of HTML, but they can quickly become difficult to track once your HTML document gets beyond a very short webpage. While we can use `id`s and `class`es to identify elements with specific names, why isn't there a way to make HTML easier to write for developers, and easier to read for browsers and search engines? That's where semantic elements come in.

Reply:
I don't really love that either. I think what I'm looking for is a little bit more of the Flatiron School "start in the middle" here. I think it works with the way you've stated it, and it's very clear, but it feels a little more documentation-ish than I'd like. However, I'm having trouble coming up with something much better, so I'm happy to let it stand.

Reply:
There were a few difficult rewrites in this one that I agree could probably be better, but not sure how. This will be a good one to circle back to in the future, with the benefit of more perspective, to see if we can continue to improve it.